searchlib/src/main/javacc/RankingExpressionParser.jj

```diff
@@ -316,7 +316,7 @@ List<ExpressionNode> args() :
     ExpressionNode argument;
 }
 {
-    ( argument = arg() { arguments.add(argument); } ( <COMMA> argument = arg() { arguments.add(argument); } )* )
+    ( ( argument = arg() { arguments.add(argument); })? ( <COMMA> argument = arg() { arguments.add(argument); } )* )
```

Comment:
This seems to allow func(,foo) to be parsed as calling func with the single argument foo.
 Is this intended ?

Reply:
No, but I don't think it matters - it would only matter if we wanted func(,foo) to mean something else than func(foo) later. Wish there was a better way to parse lists in JavaCC though.